packages/ui/src/theme/tokens/components/expander.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+  display: { value: 'block' },
+  backgroundColor: { value: '{colors.white.value}' },
+  borderRadius: { value: '{radii.medium.value}' },
+  boxShadow: {
+    value: {
+      offsetX: '0',
+      offsetY: '1px',
+      blurRadius: '10px',
+      color: '{colors.overlay.30.value}',
+    },
+  },
+  width: { value: '100%' },
+  item: {
+    marginTop: { value: '1px' },
+    boxShadow: {
+      value: {
+        offsetX: '0',
+        offsetY: '1px',
+        blurRadius: '0',
+        color: '{colors.overlay.20.value}',
+      },
+    },
+    borderStartStartRadius: { value: '0.5rem' },
+    borderStartEndRadius: { value: '0.5rem' },
+    borderEndStartRadius: { value: '0.5rem' },
+    borderEndEndRadius: { value: '0.5rem' },
+    _focus: {
+      boxShadow: {
+        value: {
+          offsetX: '0',
+          offsetY: '0',
+          blurRadius: '0',
+          spreadRadius: '2px',
+          color: '{colors.border.focus.value}',
+        },
+      },
+    },
+  },
+  trigger: {
+    height: { value: '3rem' },
```

Comment:
What happens if the content in the trigger is too long and breaks to multiple lines?